MapboxCoreNavigation/RouteController.swift

```diff
@@ -373,7 +373,18 @@ open class RouteController: NSObject {
         let wrappedCourse = location.course.wrap(min: -180, max: 180)
         let relativeAnglepointBehind = (wrappedPointBehind - wrappedCourse).wrap(min: -180, max: 180)
         let relativeAnglepointAhead = (wrappedPointAhead - wrappedCourse).wrap(min: -180, max: 180)
-        let averageRelativeAngle = pointBehindClosest.distance > 0 ? (relativeAnglepointBehind + relativeAnglepointAhead) / 2 : relativeAnglepointAhead
+        
+        var averageRelativeAngle: Double!
```

Comment:
Would be safer to define this as a constant `let averageRelativeAngle: Double` which works b/c the assignment below is exhaustive.

Reply:
Wow, did not know you could do that!